gateway/src/cluster/builder.rs

```diff
@@ -185,6 +185,17 @@ impl ClusterBuilder {
         self
     }
 
+    /// Set whether or not outgoing payloads will be ratelimited.
+    ///
+    /// Useful when running behind a proxy gateway. Running without a
+    /// functional ratelimiter **will** get you ratelimited.
+    #[allow(clippy::missing_const_for_fn)]
```

Comment:
Let's document the default value here like with other methods.
```suggestion
    /// functional ratelimiter **will** get you ratelimited.
    ///
    /// Defaults to being enabled.
    #[allow(clippy::missing_const_for_fn)]
```